These IDs cannot be set through meson options like others can. Still hardcode
them according to the 'USERIDS.md' file in the MassOS source repository.

--- a/sysusers.d/systemd-coredump.conf
+++ b/sysusers.d/systemd-coredump.conf
@@ -5,4 +5,4 @@
 #  the Free Software Foundation; either version 2.1 of the License, or
 #  (at your option) any later version.
 
-u! systemd-coredump - "systemd Core Dumper"
+u! systemd-coredump 980 "systemd Core Dumper"
--- a/sysusers.d/systemd-oom.conf
+++ b/sysusers.d/systemd-oom.conf
@@ -5,4 +5,4 @@
 #  the Free Software Foundation; either version 2.1 of the License, or
 #  (at your option) any later version.
 
-u! systemd-oom - "systemd Userspace OOM Killer"
+u! systemd-oom 978 "systemd Userspace OOM Killer"
